clash_lib/Cargo.toml

```diff
@@ -27,7 +27,7 @@ lru_time_cache = "0.11"
 hyper = { version = "0.14.28", features = ["http1","http2","client", "server", "tcp"] }
 http = { version = "1.1" }
 httparse = "1.8.0"
-h2 = "0.4.2"
+h2 = "0.4.3"
```

Comment:
are these changes picked from master branch? i don't think you should have these if you didn't modify them

Reply:
yeah, it was picked from master, changed by the dependency bot